enet.pyx

```diff
@@ -819,6 +852,8 @@ cdef class Host:
 
     cdef ENetHost *_enet_host
     cdef bool dealloc
+    cdef object _interceptCallback
+    cdef object __weakref__
```

Comment:
Tested without it, this is actually needed in order for weak references to work. Here's bit of cython documentation back up this claim: http://cython.readthedocs.io/en/latest/src/reference/extension_types.html#weak-referencing